.ci/new-github-milestone-and-publish-jar.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
```

Comment:
is there any problem to put all in existing action and shell file ?

Reply:
No, but I thought to make release scripts more specialized rather than focusing on the generalization approach. But I can revert changes if this does not look good to you.

Reply:
Instead of creating a github action, I have merged the scrips so we don't need a particular script now.

Reply:
update PR description: https://github.com/checkstyle/checkstyle/pull/12119#issue-1353314309